apps/website/src/components/show-and-tell/ShowAndTellEntryForm.tsx

```diff
@@ -578,6 +609,24 @@ export function ShowAndTellEntryForm({
           )}
         </Button>
       </div>
+
+      <ModalDialog
+        isOpen={!!previewImageUrl}
+        closeModal={closeModal}
+        title="Image Preview"
+      >
+        {previewImageUrl ? (
+          <Image
+            src={previewImageUrl}
+            alt="Form Image"
+            width={500}
+            height={500}
+            className="object-contain"
+          ></Image>
+        ) : (
+          <p>No Image to preview</p>
+        )}
```

Comment:
```suggestion
        {previewImageUrl && (
          <Image
            src={previewImageUrl}
            alt="Form Image"
            width={500}
            height={500}
            className="object-contain w-full"
          ></Image>
        )}
```
1. We shouldn't need to handle the case of no image
2. Allowing the image to fill the modal